check/check.go

```diff
@@ -210,7 +210,23 @@ func textToCommand(s string) (cmds []*exec.Cmd) {
 
 	cp := strings.Split(s, "|")
 
+	var newCP []string
+	joinNextPart := false
+
+	// Separate between pipeline and or, differentiate by | and \\|
 	for _, v := range cp {
+		if joinNextPart {
+			newCP[len(newCP)-1] = strings.Join([]string{newCP[len(newCP)-1], strings.TrimSuffix(v, "\\")}, "|")
+			joinNextPart = false
+		} else {
+			newCP = append(newCP, strings.TrimSuffix(v, "\\"))
+		}
```

Comment:
```suggestion
		vv := strings.Trim(v, "\\")
		if joinNextPart {
			newCP[len(newCP)-1] += "|" + vv
			joinNextPart = false
		} else {
			newCP = append(newCP, vv)
		}
```

Makes no difference to the execution but storing the trimmed version of `v` in a local variable `vv` is a bit easier to read, I think. 

Generally speaking `strings.Join` is more efficient but I think the benefits for readability here are worth it, wdyt?